packages/graphql/tests/tck/tck-test-files/connections/connect-or-create/connect-or-create-auth.test.ts

```diff
@@ -87,10 +87,10 @@ describe("connectOrCreate", () => {
                 CALL {
                 	WITH this0
                 	CALL apoc.util.validate(NOT(ANY(r IN [\\"admin\\"] WHERE ANY(rr IN $auth.roles WHERE r = rr))), \\"@neo4j/graphql/FORBIDDEN\\", [0])
-                MERGE (this0_genres_connectOrCreate_this1:\`Genre\` { name: $this0_genres_connectOrCreate_param0 })
+                MERGE (this0_genres_connectOrCreate_this2:\`Genre\` { name: $this0_genres_connectOrCreate_param0 })
```

Comment:
I've heard about arguments between starting counting at 0 or 1, but starting count at 2?! What's the cause of this?